lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -204,10 +204,19 @@ function dosomething_campaign_preprocess_action_page(&$vars, &$wrapper) {
   // Plan.
 
   // 1) Materials
+
+  // Build the share link here in case registering organ donors
+  $campaign_path = url(current_path(), array('absolute' => TRUE));
+  $custom_organ_share_link = $campaign_path . '?source=user/' . $vars['user']->uid;
+
   if (isset($campaign->items_needed)) {
     $materials['category'] = 'materials';
     $materials['title'] = t('Stuff You Need');
     $materials['content'] = $campaign->items_needed;
+    // if organ donation campaign, add in the share link here
+    if ($campaign->variables['register_organ_donors']) {
+      $materials['content'] .= '<ul><li><a href=' . $custom_organ_share_link . '>Your Custom Share Link</a></li></ul>';
```

Comment:
I feel like we shouldn't hyperlink these...as we only want people to copy and paste the link to share. If we hyperlink it, they have to right click/copy the link or click the link and basically go to the same page they are already on. Could we just add the full link in some `<code></code>` tags?